File: 412-fizz-buzz/412-fizz-buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        res = []
        for i in range(1, n+1):
            res.append(((i % 3 == 0)*"Fizz" + (i % 5 == 0)*"Buzz" or str(i)))
                       
        return res
    

File: 412-fizz-buzz/412-fizz-buzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {
    let res = [];
    for (let i = 1; i <= n; i++){
        if(i % 15 === 0){
            res.push("FizzBuzz");
        } else if (i % 5 === 0){
            res.push("Buzz");
        } else if (i % 3 === 0){
            res.push("Fizz");
        } else {
            res.push(i.toString());
        }
    }
    
    return res;
};